calculator/calculator.test.js

```diff
@@ -0,0 +1,49 @@
+const calculator = require('./calculator');
+
+it('adding two positive numbers', () => {
+  expect(calculator.add(1, 2)).toBe(3);
```

Comment:
[OPTIONAL]
You can try leaving less human error in your test by avoiding the hardcoded values.
Like 3 could be `1 +2`
YOu could even have variables and variables that store larger numbers as well.
like `const num2 = 2` etc.
or 
`const bigNum = 123455678`